beets/util/__init__.py

```diff
@@ -762,12 +798,12 @@ def py3_path(path):
     return os.fsdecode(path)
 
 
-def str2bool(value):
+def str2bool(value: str) -> bool:
     """Returns a boolean reflecting a human-entered string."""
     return value.lower() in ('yes', '1', 'true', 't', 'y')
 
 
-def as_string(value):
+def as_string(value: Optional[memoryview | bytes]) -> str:
```

Comment:
Need to check the call sites, but if I remember correctly this can take `Any`, not just `bytes`.

Reply:
Quite possibly, these are just the usages that occur in this file. Once it's expanded to the entirety of the project, many typings will likely have to be broadened.

Reply:
Oops wrong one

Reply:
Fixed in 4d1dec3c